src/extension.ts

```diff
@@ -59,6 +59,17 @@ export async function activate(ctx: vscode.ExtensionContext): Promise<void> {
   const storage = new Storage(output, ctx.globalState, ctx.secrets, ctx.globalStorageUri, ctx.logUri)
   await storage.init()
 
+  // Add headers from the header command.
+  axios.interceptors.request.use(async (config) => {
+    return {
+      ...config,
+      headers: {
+        ...(await storage.getHeaders()),
+        ...creds.headers,
```

Comment:
hey @code-asher  this does not compile..
I've tried to run it locally and it failed

Reply:
Yeah just realized I messed this up, fixing now.

Reply:
Pushed a fix!